src/components/Input/MonthInput.spec.js

```diff
@@ -266,6 +268,11 @@ describe('<MonthInput />', () => {
       assert.equal(calendar.props().dateVisible, dateVisible);
     });
 
+    it('should pass centerYearSelection to Calendar component', () => {
+      const calendar = component.find('MonthCalendar');
+      assert.equal(calendar.props().centerYearSelection, centerYearSelection);
+    });
```

Comment:
Is there any use for this test? It seems like it's only asserting the setup of the test.

Reply:
Not needed, just saw another test that tests prop pass-through